core/pom.xml

```diff
@@ -54,6 +54,14 @@
       <groupId>io.github.openfeign.form</groupId>
       <artifactId>feign-form</artifactId>
     </dependency>
+    <dependency>
+      <groupId>io.openapitools.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-hal</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>io.cloudevents</groupId>
+      <artifactId>cloudevents-api</artifactId>
+    </dependency>
```

Comment:
I guess you could exclude the `org.hibernate.validator:hibernate-validator` as a new  `<exclusions><exclusion>` entry here